Control/PartitionMetrics/src/ArmoniKMeter.cs

```diff
@@ -127,7 +127,11 @@ public Task StopAsync(CancellationToken cancellationToken)
     await foreach (var partition in partitionTable_.GetPartitionWithAllocationAsync(cancellationToken)
                                                    .ConfigureAwait(false))
     {
-      var metricValue = (metrics[$"armonik_{partition.PartitionId}_tasks_queued"] as Metric)!.MetricValue;
+      var metric = metrics[$"armonik_{partition.PartitionId}_tasks_queued"] as Metric;
```

Comment:
You can just use the `?` operator.
```suggestion
      var metricValue = (metrics[$"armonik_{partition.PartitionId}_tasks_queued"] as Metric)?.MetricValue;
```

Reply:
yes! that was indeed ugly